### ISSUE TITLE
Missing back button

### ISSUE THREAD

**Issue by greenwald** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#576]
Continuing off the discussion in #412 

Using version 3.0.1 (up to date as of the posting of this issue) on Mac OS X 10.11.2: I don't see the back button for navigation. Steps to reproduce: go to the apple store, purchase computer, install this app.

The same is actually true of version 3.0.1-1 on Ubuntu 14.04.4.

So either there's a bug in the software that prevents the back button showing up. Or there's a design flaw that's making it hard for me to find.

**Comment by MarshallOfSound** [comment]
The back button only appears if there is something to go "back" to.  I.e. it won't appear when you open the app, you have to navigate somewhere for it to appear. 

![back](https://cloud.githubusercontent.com/assets/6634592/13749573/399d489a-ea56-11e5-8e10-ebc04168191a.PNG)

*P.S. Those steps to reproduce are shocking* :cactus:

**Comment by greenwald** [comment]
I understand. It doesn't show up after I navigate away from the initial screen.

**Comment by MarshallOfSound** [comment]
Just testing on my mac that is also on `10.11.2` and it pops up just fine.  Can you do a screencast or something to show what you are doing?

**Comment by mspl13** [comment]
I also can not confirm it for Ubuntu 14.04.4 and 3.0.1. Works fine for me (see screenshot below).
![screenshot from 2016-03-14 23 20 00](https://cloud.githubusercontent.com/assets/8393513/13761857/65b6abee-ea3b-11e5-9a50-05fc4849b00f.png)

**Comment by greenwald** [comment]
Don't know what else to say: using version 3.0.1 downloaded as a release from the website I don't see the back button. Not on the Mac, not on Ubuntu.

**Comment by mspl13** [comment]
Did you clean install it in 3.0.1 or updated it from an older version?

**Comment by greenwald** [comment]
Both, in fact. On the Mac, that is. On Ubuntu I updated.

**Comment by MarshallOfSound** [comment]
That is REALLY strange, I'm going to keep testing my devices to see if I can get one to break

**Comment by SolarPolarMan** [comment]
I just realise that this affects me aswell. On eOS 0.3.2. (Ubuntu 14.04)

**Comment by MarshallOfSound** [comment]
I don't understand how this can affect some people and not others....

Until I get access to a device / account that has this issue there isn't much I can do to diagnose it

**Comment by blehn** [comment]
I would also propose using backspace and/or ctrl+left arrow to trigger back navigation

**Comment by dansauve** [comment]
Confirming that this is also a problem for me on my Windows 7 PC at work running v3.0.1. I can't post a screencast from this machine either but suffice to say:

- Upgraded originally, but still no back arrow after uninstall/reinstall 
- Backspace does nothing
- Cntrl-left arrow goes to the start of the song, not 'Back'

**Comment by MarshallOfSound** [comment]
I don't get why people keep mentioning the fact that backspace and Ctrl+Left don't work.  They have not been implemented and have never been implemented.  Do people think they will just suddenly start working or something????

In other news I still can't replicate the back button not appearing :cry: If anyone can replicate this in a dev enivironment and can figure out why I'll give the 3 :cactus: emoji :+1:

**Comment by MarshallOfSound** [comment]
It was a long road but I finally managed to replicate the bug :cactus: 

I think it's fixed, if anyone is running betas / master they can check it out otherwise hold out till the next release

**Comment by dansauve** [comment]
Glad to hear it is likely fixed, I'll watch for the next release.

Just FYI on the backspace/control-left mention, someone else mentioned in the thread that it might work so I was just confirming that it doesn't. Backspace works in the web browser of course, so it might be nice to consider as a shortcut in your app as well.

**Comment by blehn** [comment]
@MarshallOfSound re: backspace, ctrl/cmd-left — nobody said it was broken, or a bug. Given the lack of browser back button (and the fact that this is how browsers behave), I thought it was a reasonable proposal. But hey, it's your software.

**Comment by MarshallOfSound** [comment]
@blehn I have since implemented the backspace button but my point was that people were reporting things as "not working" as if it was a bug, when it shouldn't be working  in the first place.  I was trying to explain the difference between a bug and a feature request :+1: :cactus: 

TLDR: backspace is implemented

**Comment by greenwald** [comment]
Just got the next release... back button shows up (on the Mac). Thanks!

**Comment by SolarPolarMan** [comment]
Fixed (ubuntu)
